libdnf/dnf-package.cpp

```diff
@@ -131,6 +131,13 @@ dnf_package_get_filename(DnfPackage *pkg)
         }
     }
 
+    /* remove file:// protocol */
+    if (g_str_has_prefix(priv->filename, "file://")){
```

Comment:
Similar to https://github.com/rpm-software-management/createrepo_c/pull/124, the check likely should be for `file:///`.